composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "myparcelnl/sdk",
-    "version": "v3.1.8",
+    "version": "v3.2.0",
     "description": "This package is designed to send and receive data from MyParcel by means of an API.",
     "homepage": "https://www.myparcel.nl",
```

Comment:
Ik mis nog documentatie in README.md
Mag klein en beknopt zijn hoor.

Reply:
Done